src/plugins/lightscript.js

```diff
@@ -134,40 +134,45 @@ pp.parseObjectComprehension = function(node) {
   return this.finishNode(node, "ObjectComprehension");
 };
 
-// c/p parseBlock
+pp.finishWhiteBlock = function(node) {
+  if (!node.body.length) {
+    this.unexpected(node.start, "Expected an Indent or Statement");
+  }
+
+  this.addExtra(node, "curly", false);
+  return this.finishNode(node, "BlockStatement");
+};
 
-pp.parseWhiteBlock = function (allowDirectives?, isExpression?) {
+pp.parseInlineWhiteBlock = function(node) {
+  if (this.state.type.startsExpr) return this.parseMaybeAssign();
+  // oneline statement case
+  node.body = [this.parseStatement(true)];
```

Comment:
Are you sure about this? The node gets turned into a `BlockStatement` by `finishWhiteBlock`, and babel-types says those have to have bodies that are arrays:

```
defineType("BlockStatement", {
  builder: ["body", "directives"],
  visitor: ["directives", "body"],
  fields: {
    directives: {
      validate: chain(assertValueType("array"), assertEach(assertNodeType("Directive"))),
      default: [],
    },
    body: {
      validate: chain(assertValueType("array"), assertEach(assertNodeType("Statement"))),
    },
  },
  aliases: ["Scopable", "BlockParent", "Block", "Statement"],
});
```